sample/Pizzeria.Store.WebClient/States/Orders/OrdersEffects.cs

```diff
@@ -1,48 +1,64 @@
 using System.Diagnostics.CodeAnalysis;
+using Correlate;
 using Fluxor;
+using Lewee.Blazor.Fluxor;
+using Microsoft.AspNetCore.Components;
+using Microsoft.Extensions.Logging;
 using Pizzeria.Store.WebClient.Services;
 using Pizzeria.Store.WebClient.States.Orders.Actions;
 
 namespace Pizzeria.Store.WebClient.States.Orders;
 
-public class OrdersEffects
+public class OrdersEffects : RequestEffects<OrdersState, StartOrderAction, StartOrderSuccessAction, StartOrderFailureAction>
 {
     private readonly IPizzeriaApiClient apiClient;
+    private readonly NavigationManager navigationManager;
 
-    public OrdersEffects(IPizzeriaApiClient apiClient)
+    public OrdersEffects(
+        IState<OrdersState> state,
+        ICorrelationContextAccessor correlationContextAccessor,
+        ILogger<OrdersEffects> logger,
+        IPizzeriaApiClient apiClient,
+        NavigationManager navigationManager)
+        : base(state, correlationContextAccessor, logger)
     {
         this.apiClient = apiClient;
+        this.navigationManager = navigationManager;
     }
 
     [EffectMethod]
     [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1313:Parameter '_' should begin with lower-case letter", Justification = "Underscore is the standard discard pattern for unused parameters")]
-    public async Task OnStartOrderAsync(StartOrderAction _, IDispatcher dispatcher)
+    [SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1202:'public' members should come before 'protected' members", Justification = "Fluxor EffectMethod must be public")]
+    public Task OnStartOrderCompletedAsync(StartOrderCompletedAction _, IDispatcher __)
+    {
+        this.navigationManager.NavigateTo("/order");

```

Comment:
Can we have a static `Routes` class with constants for all the routes?

It can be used in places like this and also razor-files.